app/controllers/conversations_controller.rb

```diff
@@ -34,4 +36,12 @@ def signed_in_flash
       redirect_to users_path
     end
   end
+
+  def participant?
+    @conversation = Conversation.find(params[:id])
+    unless (current_user.id == @conversation.recipient_id) || (current_user.id == @conversation.sender_id)
+      redirect_to users_path
+      flash[:alert] = 'You are not authorized to enter this chat'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.